Gems/EMotionFX/Code/EMotionFX/Source/NodeGroup.h

```diff
@@ -30,38 +30,19 @@ namespace EMotionFX
      * might contain incorrect or even uninitialized data.
      */
     class EMFX_API NodeGroup
-        : public BaseObject
     {
     public:
         AZ_CLASS_ALLOCATOR_DECL
-        /**
-         * The default creation method.
-         * This does not assign a name and there will be nodes inside this group on default.
-         * Also the default enabled state is set to true.
-         */
-        static NodeGroup* Create();
-
-        /**
-         * Extended creation.
-         * @param groupName The name of the group. Please keep in mind that it is not allowed to have two groups with the same name inside an Actor.
-         * @param enabledOnDefault Set to true (default) when the nodes inside this group should be enabled on default.
-         */
-        static NodeGroup* Create(const char* groupName, bool enabledOnDefault = true);
 
-        /**
-         * Another extended constructor.
-         * @param groupName The name of the group. Please keep in mind that it is not allowed to have two groups with the same name inside an Actor.
-         * @param numNodes The number of nodes to create inside the group. This will have all uninitialized values for the node indices in the group, so be sure that you
-         *                 set them all to some valid node index using the NodeGroup::SetNode(...) method. This method automatically calls the SetNumNodes(...) method.
-         * @param enabledOnDefault Set to true (default) when the nodes inside this group should be enabled on default.
-         */
-        static NodeGroup* Create(const char* groupName, uint16 numNodes, bool enabledOnDefault = true);
+        NodeGroup(const AZStd::string& groupName = {}, uint16 numNodes = 0, bool enabledOnDefault = true);
```

Comment:
Repeat: The groupName parameter could be accepted by value and moved into the member variable